applications/GeoMechanicsApplication/custom_constitutive/incremental_linear_elastic_law.h

```diff
@@ -21,27 +21,27 @@ namespace Kratos
 class ConstitutiveLawDimension;
 
 /**
- * @class GeoLinearElasticPlaneStrain2DLaw
+ * @class GeoIncrementalLinearElasticLaw
  * @ingroup GeoMechanicsApplication
- * @brief This class defines a small deformation linear elastic constitutive model for plane strain cases
+ * @brief This class defines a small deformation linear elastic constitutive model for plane strain and 3D cases
  * @author Vahid Galavi
```

Comment:
I think this could be re-written a bit more (as you already started doing so): I'd propose something like:
`This class defines an incremental linear elastic constitutive model for plane strain and 3D cases.`

I don't think that 'small deformation' is really accurate (or has anything to do with the linear elastic model), but please correct me if I'm wrong (tagging you @WPK4FEM, to hear your opinion on this as well)

Reply:
To stress the incremental nature of the implementation, we could replace "small deformation" with "incremental". The formulation  can be used both for small deformation and large deformation.

Reply:
replaced `small deformation` with `incremental`